MSAL/test/integration/native_auth/end_to_end/MSALNativeAuthEndToEndBaseTestCase.swift

```diff
@@ -129,4 +171,15 @@ class MSALNativeAuthEndToEndBaseTestCase: XCTestCase {
             return Constants.clientIdEmailCodeAttributesKey
         }
     }
+    
+    private func getCustomTenantSubdomain(tenantName: String, tenantId: String, format: AuthorityURLFormat) -> String {
```

Comment:
Could you please call this method `getAuthotityURLString` ? It seems more accurate.
```suggestion
    private func getAuthotityURLString(tenantSubdomain: String, tenantId: String, format: AuthorityURLFormat) -> String {
```
And could you please use `tenantSubdomain` instead of `tenantName`. They are two different properties

Reply:
Done.